vm.go

```diff
@@ -5591,6 +5592,10 @@ func (r *getPrivateRefId) exec(vm *vm) {
 }
 
 func (s valueStack) MemUsage(ctx *MemUsageContext) (memUsage uint64, err error) {
+	return valuesMemUsage(s, ctx)
+}
+
+func valuesMemUsage(s []Value, ctx *MemUsageContext) (memUsage uint64, err error) {
 	for _, val := range s {
```

Comment:
[nit] I just noticed we have been missing accounting for `SizeEmptySlice`, it won't make a ton of difference though.

Reply:
Ah good catch! We could do something like this:

```
func valuesMemUsage(s []Value, ctx *MemUsageContext) (memUsage uint64, err error) {
	if len(s) == 0 {
		memUsage += SizeEmptySlice
	}

	for _, val := range s {
```

Reply:
You can remove the if statement, we account for it no matter what as the data structure holding the values.

Reply:
Yeah I just noticed that :D we were missing it always 😬